Source: Elias Pipping <pipping@exherbo.org>
Upstream: http://llvm.org/bugs/show_bug.cgi?id=6713
Reason: Respect variables set by configure

---
 Makefile.config.in |   26 +++++++++++++++++++++-----
 1 files changed, 21 insertions(+), 5 deletions(-)

diff --git a/Makefile.config.in b/Makefile.config.in
index 3961cc9..b96d6ad 100644
--- a/Makefile.config.in
+++ b/Makefile.config.in
@@ -61,6 +61,23 @@ endif
 
 prefix          := @prefix@
 PROJ_prefix     := $(prefix)
+exec_prefix     := @exec_prefix@
+bindir          := @bindir@
+PROJ_bindir     := $(bindir)
+libdir          := @libdir@
+PROJ_libdir     := $(libdir)
+datadir         := @datadir@
+PROJ_datadir    := $(datadir)
+docsdir         := @docdir@
+PROJ_docsdir    := $(docsdir)
+etcdir          := @sysconfdir@/llvm
+PROJ_etcdir     := $(etcdir)
+includedir      := @includedir@
+PROJ_includedir := $(includedir)
+infodir         := @infodir@
+PROJ_infodir    := $(infodir)
+mandir          := @mandir@
+PROJ_mandir     := $(mandir)
 PROJ_VERSION    := $(LLVMVersion)
 else
 ifndef PROJ_SRC_ROOT
@@ -81,11 +98,6 @@ endif
 PROJ_SRC_DIR := $(call realpath, $(PROJ_SRC_ROOT)/$(patsubst $(PROJ_OBJ_ROOT)%,%,$(PROJ_OBJ_DIR)))
 prefix          := $(PROJ_INSTALL_ROOT)
 PROJ_prefix     := $(prefix)
-ifndef PROJ_VERSION
-PROJ_VERSION := 1.0
-endif
-endif
-
 PROJ_bindir     := $(PROJ_prefix)/bin
 PROJ_libdir     := $(PROJ_prefix)/lib
 PROJ_datadir    := $(PROJ_prefix)/share
@@ -94,6 +106,10 @@ PROJ_etcdir     := $(PROJ_prefix)/etc/llvm
 PROJ_includedir := $(PROJ_prefix)/include
 PROJ_infodir    := $(PROJ_prefix)/info
 PROJ_mandir     := $(PROJ_prefix)/share/man
+ifndef PROJ_VERSION
+PROJ_VERSION := 1.0
+endif
+endif
 
 # Determine if we're on a unix type operating system
 LLVM_ON_UNIX:=@LLVM_ON_UNIX@
-- 
1.7.7

